dsc/workflows/base/__init__.py

```diff
@@ -313,27 +313,35 @@ def process_sqs_queue(self) -> list[dict]:
         sqs_client = SQSClient(
             region=CONFIG.aws_region_name, queue_name=self.output_queue
         )
+
         items = []
         for sqs_message in sqs_client.receive():
             try:
-                item_identifier, message_body = sqs_client.process_result_message(
+                item_identifier, result_message = sqs_client.process_result_message(
```

Comment:
This is technically just the message body that's returned by `process_result_message`, the result message contains the message attributes as well (where the only thing we care about currently is the `item_identifier` which we're already extracting)